api/v1alpha1/observatorium_types.go

```diff
@@ -45,6 +45,9 @@ type ObservatoriumSpec struct {
 	// Loki
 	// +optional
 	Loki *LokiSpec `json:"loki,omitempty"`
+	// NodeSelector applies to all components
+	// +optional
+	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
```

Comment:
Is this the data type used in Kubernetes too?

Reply:
Yes. see here https://pkg.go.dev/k8s.io/api@v0.18.3/core/v1#PodSpec